packages/app/components/Nav/NavBar.tsx

```diff
@@ -309,5 +303,49 @@ export default function NavBar({ courseId }: NavBarProps): ReactElement {
         </Modal>
       )}
     </>
-  ) : null;
+  ) : (
+    <>
+      <NavBG />
+      <Nav>
+        <LogoContainer>
+          <Logo>
+            {profile?.organization && (
+              <a href="/courses">
```

Comment:
The settings page is a global page, so I think it makes for a logo to redirect to the "main" page a.k.a courses. For individual courses, I think there should be a focus on redirecting to the main page of course instead of courses